devops/docker/smqtk_iqr_playground/README.md

```diff
@@ -73,16 +72,14 @@ nothing to process/ingest. The ``-b`` option must also be given in order to
 trigger model building.
 
 
-RESTful service
-^^^^^^^^^^^^^^^
+## RESTful service
```

Comment:
I believe this one should be `###` to correspond to a Sphinx subsubsection (`^^`).